.github/pull_request_template.md

```diff
@@ -2,6 +2,10 @@ Closes #
 
 Changes proposed in this pull request:
 
-*
-*
-* ...
+- 
+
+#### Instructions for Reviewers
+
+_[All PRs] - Confirm PR template filled_  
+_[Feature Branches] - Review code_  
+_[Develop Merges] - Check version_  
```

Comment:
Does this include Develop -> Master, include story numbers of whats being merged?